examples/gltf/src/main.rs

```diff
@@ -0,0 +1,156 @@
+use std::sync::Arc;
+
+fn load_gltf(
+    renderer: &rend3::Renderer,
+    path: &'static str,
+) -> (rend3::datatypes::MeshHandle, rend3::datatypes::MaterialHandle) {
+    let (doc, datas, _) = gltf::import(path).unwrap();
+    let mesh_data = doc.meshes().next().expect("no meshes in data.glb");
+
+    let primitive = mesh_data.primitives().next().expect("no primitives in data.glb");
+    let reader = primitive.reader(|b| Some(&datas.get(0)?.0[b.index()..b.length()]));
+    let mut normals = reader.read_normals();
+    let mut uv = reader.read_tex_coords(0).map(|tc| tc.into_f32());
+    let mut mesh = rend3::datatypes::Mesh {
+        vertices: reader
+            .read_positions()
+            .unwrap()
+            .map(|pos| rend3::datatypes::ModelVertex {
```

Comment:
Honestly, this would be better as a set of zips, this code is pretty dense as it is.

Reply:
It's less fault tolerant as a set of zips. This wouldn't actually render anything at all that way because this data doesn't have texcoords.